jetty-http2/http2-server/pom.xml

```diff
@@ -15,6 +15,32 @@
     <manual.test.port>28888</manual.test.port>
   </properties>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.mortbay.jetty</groupId>
+        <artifactId>h2spec-maven-plugin</artifactId>
+        <configuration>
+          <mainClass>org.eclipse.jetty.http2.server.H2SpecServer</mainClass>
+          <skip>${skipTests}</skip>
+          <junitPackage>org.eclipse.jetty.h2spec</junitPackage>
+          <skipNoDockerAvailable>true</skipNoDockerAvailable>
```

Comment:
this attribute makes the plugin not failing if docker not available but print a warning